src/pdu_header.cpp

```diff
@@ -0,0 +1,71 @@
+#include <cfdp/pdu_header.hpp>
+
+#include "utils.hpp"
+
+/*
+    Without the last 3 fields, the rest of the PDU header has constant
+    size. Calculating all used bits in order:
+    sizeBits = 3 + 1 + 1 + 1 + 1 + 1 + 16 + 1 + 3 + 1 + 3 = 32
+*/
+inline constexpr uint16_t CONST_HEADER_SIZE_BYTES      = 32 / 8;
+inline constexpr uint8_t VERSION_POSITION              = 5;
```

Comment:
I'm not sure if it's a good idea to set a const position from right side of a particular byte